### PR TITLE
removed "local" and fixed docker-compose.yml

### PR DESCRIPTION
**Pull request by zoekulphong**
Since this can be done either remotely or locally, saying "local machine" could be confusing (at least for me)

The dip run bash command gave an error when the first three lines of the docker-compose.yml file were in the actual file, so I moved them out, you may want to reformat it if needed

### CODE REVIEW

File: rails-new.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ Instead of having to manage the versions of multiple different dependancies for
...
 In your new directory make a `docker-compose.yml` file and paste in:
 
-```ruby
-# docker-compose.yml
+docker-compose.yml
 --------------------
+```ruby
 
 version: "3"
 services:
```
Comment (janzenisaac) [inline rails-new.md:23]: So we are using the format of 
```
# docker-compose.yml
--------------------

Content here below
```

to show that you are in a file titled docker-compose.yml, and below this dotted line you will add **x** content. Maybe we can add a small blurb in the introduction explaining this, but you will see this same method time and time again throughout the book

**Comment by nathancolgate** [comment]
Interesting catch, @zoekulphong . We intentionally put those in as helpers to indicate the specific location of the file we are showing. But if you copy-n-paste the whole thing, it kind of falls apart. Maybe we can come up with a better paradigm for this.

Once we do, we will need to update the book in dozens of places.

**Comment by nathancolgate** [comment]
Trying other options:

<small>docker-compose.yml</small>

```ruby
5.times do { }
```

<figure>docker-compose.yml</figure>

```ruby
5.times do { }
```

<figure><strong>docker-compose.yml</strong></figure>

```ruby
5.times do { }
```

<figure><strong><code>docker-compose.yml</code></strong></figure>

```ruby
5.times do { }
```

Ooo... that last one looks slick. The markdown is:

```
<figure><strong><code>docker-compose.yml</code></strong></figure>
```

**Comment by nathancolgate** [comment]
@zoekulphong we _really_ like that last look using the `<figure><strong><code>` syntax. As part of this PR, could you please update the book to use that everywhere there is an example? Thanks!

**Comment by zoekulphong** [comment]
@nathancolgate @janzenisaac I like that syntax too! Will do.

**Review by janzenisaac — APPROVED** [review]
Looks great!

**Review by nathancolgate — APPROVED** [review]

**Comment by nathancolgate** [comment]
@zoekulphong Once you merge a branch, you are welcome to delete the branch from origin!

**Pull request merged**